config/ros_control_controllers.yaml

```diff
@@ -0,0 +1,210 @@
+# Publish all joint states -----------------------------------------------------
+joint_state_controller:
+  type: joint_state_controller/JointStateController
+  publish_rate: 500
```

Comment:
This is way too fast to publish over ROS. I suggest decreasing this to 50 - 100 Hz.